jenkins/jenkinspipeline-build-and-deploy-job.txt

```diff
@@ -10,13 +10,29 @@ pipeline {
 	}
     agent any
     stages {
-        stage('Checkout') {
+        stage("Checkout") {
             steps {
                 // Get some code from a GitHub repository
                 echo "Getting code from main"
                 git branch: "${BRANCH_NAME}", url: "https://github.com/bcgov/moh-hni-esb"
             }
         }
+ 		stage("Package hn-common library") {
+            steps {
+                dir("hn-common") {
+                    echo "Running: mvn install"
+                    sh "mvn install"
+                }	
+            }
+        }
```

Comment:
Unrelated but eventually we should have a separate job to build and install hn-common and deploy it to Artifactory (or whatever Repository OpenShift provides).

Reply:
Agree, this one is not using antifactory because Openshift Jenkins can not access behind firewall BC health  artifactory. I can create a story/task to add new job for build & install hn-client. And update current jobs as well.